inst/examples/dashboard/style.scss

```
@@ -1,16 +1,114 @@
 @import "about_section";
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:ital,wght@0,400;0,500;0,600;1,400;1,500;1,600&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Maven+Pro:wght@500;600;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;1,400;1,500&display=swap');
 
 :root {
-  --AppsilonBlue: #0099F9;
-  --AppsilonBlue-weak-2: #80CCFC;
-  --AppsilonMint: #1AFFD5;
-  --AppsilonMint-weak-2: #C2FFF4;
-  --AppsilonPurple-weak-1: #994B9D;
-  --AppsilonPurple-weak-2: #BB87BE;
-  --AppsilonRose-weak-1: #FF8692;
-  --AppsilonYellow-weak-1: #FDE16E;
+  --AppsilonBlue:     #0099F9;
+  --AppsilonNavi:     #15354A;
+  --AppsilonIce:      #A6D5E0;
+  --AppsilonMint:     #1AFFD5;
+  --AppsilonYellow:   #FCD73D;
+  --AppsilonRose:     #FF364A;
+  --AppsilonPurple:   #770F7C;
+  --AppsilonGray:     #6E757B;
+  --AppsilonLightGray:#8A8E91;
+  --AppsilonAsh:      #B3B8BA;
+  --AppsilonSnow:     #FFFFFF;
```

Let's not align values like this; it's painful to maintain: one name change might require spacing adjustment in multiple lines (easy to forget / make a mistake, larger nonmeaningful diff - harder to review).

```suggestion
  --AppsilonBlue: #0099F9;
  --AppsilonNavi: #15354A;
  --AppsilonIce: #A6D5E0;
  --AppsilonMint: #1AFFD5;
  --AppsilonYellow: #FCD73D;
  --AppsilonRose: #FF364A;
  --AppsilonPurple: #770F7C;
  --AppsilonGray: #6E757B;
  --AppsilonLightGray: #8A8E91;
  --AppsilonAsh: #B3B8BA;
  --AppsilonSnow: #FFFFFF;
```

---

I can make this change, but again this part of code has been taken from "Kitchen UI Sink" example.

---

Ok, let's leave it as is :+1: